src/components/app.js

```diff
@@ -32,6 +32,10 @@ var appContainer = document.getElementById('app');
 
 const store = configureStore({});
 
+history.listen(() => {
+  window.scrollTo(0, 0);
+});
+
```

Comment:
But when you navigate back (previous page) it would be good to keep the position, dont you think?

Reply:
Yeah indeed. The back button still does that though, for me at least. I think it's a standard browser behaviour somehow. Not sure how it is implemented to know the difference here, but it seems to work.